modules/microshift-storage-creating-vol-snapshot.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-volume-snapshotting_{context}"]
+= Creating a volume snapshot
+
+To create a snapshot of a {product-title} storage volume, you must first configure {op-system-ostree} and the cluster. In the following example procedure, the pod that the source volume is mounted to is deleted. Deleting the pod prevents data continuing to be written to it during snapshot creation. If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
+
+[NOTE]
+====
+Volume snapshots are located on the same devices as the original data. To use the volume snapshots as backups, move the snapshots to a secure location.
+====
+
+.Prerequisites
+
+. User has root access to a {product-title} cluster.
+. A {product-title} cluster is running.
+. A device class defines an LVM thin-pool.
+. A `volumeSnapshotClass` specifies `driver: topolvm.io`.
+. Any workload attached to the source PVC is paused or deleted. This helps avoid data corruption.
+
+[IMPORTANT]
+====
+All writes to the volume must be halted while you are creating the snapshot. If you do not halt writes, your data might be corrupted.
+====
+
+.Procedure
+
+. Delete the pod to ensure that no data is written to the volume during snapshotting by running the following command:
++
+[source, terminal]
+----
+$ oc delete my-pod
+----
+
+. Optional: If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
```

Comment:
This could just be a note as part of step 1, since it's an alternative to that step if they are using their own workload instead of our example.